lib/screens/calendar/calendar.dart

```diff
@@ -673,10 +645,8 @@ class _EntriesSlot extends StatelessWidget {
                         crossAxisAlignment: CrossAxisAlignment.start,
                         children: [
                           Text(
-                            (DateTime.parse(entriesList[i].end)
-                                        .isAfter(DateTime.now()) &&
-                                    DateTime.parse(entriesList[i].end)
-                                        .isBefore(DateTime.now()))
+                            isHappeningNow(
+                                    entriesList[i].end, entriesList[i].start)
                                 ? 'Now'
                                 : '${DateTime.parse(entriesList[i].start).toIso8601String().substring(11, 16)}',
```

Comment:
use DateFormat(format).format(time) so that it's more obvious what's going on here

Reply:
^this is not done

Reply:
fixed